target/linux/ramips/dts/RB750Gr3.dts

```diff
@@ -20,6 +20,12 @@
 	gpio-leds {
 		compatible = "gpio-leds";
 
+		pwr {
```

Comment:
please use this LED in target/linux/ramips/base-files/etc/diag.sh and remove the default-state = "on". If the led is enabled by the bootloader, use default-state = "keep".